zipkin-storage/cassandra/src/test/java/zipkin/storage/cassandra/IndexerTest.java

```diff
@@ -1,87 +0,0 @@
-/**
- * Copyright 2015-2016 The OpenZipkin Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the License
- * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
- * or implied. See the License for the specific language governing permissions and limitations under
- * the License.
- */
-package zipkin.storage.cassandra;
-
-import com.google.common.collect.ImmutableSetMultimap;
-import com.google.common.collect.Maps;
-import java.util.concurrent.ConcurrentMap;
-import org.junit.Test;
-import zipkin.internal.Pair;
-
-import static org.assertj.guava.api.Assertions.assertThat;
-import static zipkin.storage.cassandra.Tables.SERVICE_NAME_INDEX;
-import static zipkin.storage.cassandra.Tables.SERVICE_SPAN_NAME_INDEX;
-
-public class IndexerTest {
```

Comment:
Indexer's still alive (as far as I can tell), so we shouldn't delete this until we delete that.